### PR TITLE
Defer css load 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@projectmushroom please review 

This is an attempt to defer load smart styles.
Good improvements but to keep in mind there is not much content on my local site.
QA Site
screen shot 2018-02-13 at 16 32 49

Local Site:
screen shot 2018-02-13 at 16 29 32

Issues, 
How can we optimize and include Django template compress hashing to those files?

File: gem/static/js/springster.js [hunk 1]
```diff
@@ -32,17 +32,16 @@
     var content = document.getElementById("content-wrapper");
     var menuList = document.getElementById("nav-list");
 
-    //WHY IS THIS BG APPLIED
+    //REASONS FOR BACKGROUND
```
Comment (projectmushroom) [inline gem/static/js/springster.js]: with the Rubber-Band / Elastic Scroll effect on some smartphone browsers, this sets the background color when scrolling

Comment (projectmushroom) [reply gem/static/js/springster.js]: also reinitialises when changing from portrait to landscape

Comment (projectmushroom) [reply gem/static/js/springster.js]: ezgif-4-db940d22f4

File: gem/templates/springster/patterns/basics/surveys/surveys-modules/form.html [hunk 2]
```diff
@@ -21,6 +21,9 @@
                   <input name="{{ field.name }}" id="value-{{ field.id_for_label }}-{{ forloop.counter }}" type="checkbox" value="{{ value }}" {% if field.value.0 == value %}checked="checked"{% endif %}/>
                   <label for="value-{{ field.id_for_label }}-{{ forloop.counter }}">{{ text|capfirst }}</label>
               </div>
+              {{field.field.choices}}<br/><br/>
+            -->{{ field.name }}<br/><br/>
```
Comment (codiebeulaine) [inline gem/templates/springster/patterns/basics/surveys/surveys-modules/form.html]: This should not be here

File: gem/templates/springster/patterns/basics/surveys/surveys-modules/form.html [hunk 3]
```diff
@@ -21,6 +21,9 @@
...
+              ==>{{ field.id_for_label }}<br/><br/>
```
Comment (codiebeulaine) [inline gem/templates/springster/patterns/basics/surveys/surveys-modules/form.html]: This should not be here

File: gem/templates/springster/base.html [hunk 4]
```diff
@@ -29,61 +29,56 @@
...
+    {% compress css %}
+      <link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}css/{{ ENV }}/05_no-script-state.css" media="all">
+    {% endcompress %}
     </noscript>
-    {% molo_pwa_meta %}
-
+    {% compress js inline %}
     <script type="text/javascript">
+    var mycss_file = {{smartFile}}
+    console.log('Hello world',mycss_file);
```
Comment (projectmushroom) [inline gem/templates/springster/base.html]: Helllo world!!

**Pull request merged**